vendor/krew/Makefile

```diff
@@ -0,0 +1,33 @@
+## Package template for non-github tarball
+
+## Package details 
+export VENDOR = kubernetes-sigs
+export PACKAGE_NAME = krew
+export PACKAGE_REPO_NAME = krew
+export DOWNLOAD_URL = $(PACKAGE_REPO_URL)/releases/download/v$(PACKAGE_VERSION)/krew.tar.gz
+
+## APK build specific requirements
+export APK_BUILD_TEMPLATE = APKBUILD.github-binary
+## Uncomment only if you need additional libraries
+#export APKBUILD_DEPENDS += libc6-compat musl
+
+include ../../tasks/Makefile.package
+include ../../tasks/Makefile.apk
+
+## Package specific install code
```

Comment:
In the future, you could write:
```suggestion
## Package specific install code
install: KREW_ROOT=$(INSTALL_PATH)
```

Then you can get rid of all the `&&` stuff because make automatically exits non-zero if any of the commands fail.